index.js

```diff
@@ -94,7 +95,8 @@ module.exports = {
 
     this.ui.writeLine('Generating files needed by Storybook');
 
-    fs.writeFileSync(previewHeadFilePath, previewHead)
+    fs.mkdirSync(previewHeadDirectory, { recursive: true });
+    fs.writeFileSync(previewHeadFilePath, previewHead);
```

Comment:
This creates the `.storybook` directory, if it doesn't exist, before creating `preview-head.html`